.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+
+jobs:
+  dist:
+    name: Build wheel and .tar files for release
```

Comment:
is this actually building `.tar` files? i guess it won't do anything yet since there is no python project in this repo.

Reply:
can you update this to reflect what it's actually doing?